articles/quickstart/webapp/nextjs/01-login.md

```diff
@@ -143,25 +187,45 @@ The profile information is available through the `user` property exposed by the
 
 ```jsx
 'use client';
-
-import { useUser } from '@auth0/nextjs-auth0/client';
-
-export default function ProfileClient() {
-  const { user, error, isLoading } = useUser();
-
-  if (isLoading) return <div>Loading...</div>;
-  if (error) return <div>{error.message}</div>;
-
+import React from 'react';
+import { Row, Col } from 'reactstrap';
+import { useUser } from '@auth0/nextjs-auth0';
+import Loading from '../../components/Loading';
+import Highlight from '../../components/Highlight';
+
+export default function Profile() {
+  const { user, isLoading } = useUser();
   return (
-    user && (
-      <div>
-        <img src={user.picture} alt={user.name} />
-        <h2>{user.name}</h2>
-        <p>{user.email}</p>
-      </div>
-    )
+    <>
+      {isLoading && <Loading />}
+      {user && (
+        <>
+          <Row className="align-items-center profile-header mb-5 text-center text-md-left" data-testid="profile">
+            <Col md={2}>
+              <img
+                src={user.picture}
+                alt="Profile"
+                className="rounded-circle img-fluid profile-picture mb-3 mb-md-0"
+                decode="async"
+                data-testid="profile-picture"
+              />
+            </Col>
+            <Col md>
+              <h2 data-testid="profile-name">{user.name}</h2>
+              <p className="lead text-muted" data-testid="profile-email">
+                {user.email}
+              </p>
+            </Col>
+          </Row>
+          <Row data-testid="profile-json">
+            <Highlight>{JSON.stringify(user, null, 2)}</Highlight>
+          </Row>
+        </>
+      )}
+    </>
```

Comment:
Should we realy care about all this reactstrap stuff in the quickstart? I think it clutters things unneccesarily.